### PR TITLE
Verkholiak so many changes...)

### PR DESCRIPTION
**Pull request by AndriY2389**

### CODE REVIEW

File: src/main/java/com/softserve/demo/util/ContractMaker.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        content.addImage(getQRImage(providerSignature, PROVIDER_SIGNATURE_X_POSITION));
+    }
+
+    private Image getQRImage(String userSignature, Float absoluteX) throws BadElementException {
+        Image qrUserSignature = new BarcodeQRCode(userSignature, QR_SIZE, QR_SIZE, null).getImage();
+        qrUserSignature.setAbsolutePosition(absoluteX, ABSOLUTE_Y_POSITION);
+        return qrUserSignature;
+    }
+
+    private void setFields(Order order, AcroFields fields) throws IOException, DocumentException {
```
Comment (VolodymyrMoskalyuk) [inline src/main/java/com/softserve/demo/util/ContractMaker.java]: I think it`s better called this method setContractFields or setContractData

Comment (AndriY2389) [reply src/main/java/com/softserve/demo/util/ContractMaker.java]: I fixed it.

**Review by AnastasiyaStr — APPROVED** [review]

**Review by serhiylev — APPROVED** [review]

File: src/main/java/com/softserve/demo/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -86,4 +134,15 @@ private void emailAboutOrder(Order order, String header, String message) {
...
     }
+
+    private boolean isOrderApproved(Order order) {
+        if (order.getCustomerApproved() == null || order.getProviderApproved() == null) {
+            return false;
+        }
+        return (isApproved(order.getCustomerApproved()) && isApproved(order.getProviderApproved()));
+    }
+
+    private boolean isApproved(String x) {
```
Comment (SilichAlex) [inline src/main/java/com/softserve/demo/service/impl/OrderServiceImpl.java]: String x?

Comment (AndriY2389) [reply src/main/java/com/softserve/demo/service/impl/OrderServiceImpl.java]: I fixed it.

**Review by AnastasiiaHanuliak — APPROVED** [review]

**Pull request merged**